UM/PluginRegistry.py

```diff
@@ -624,6 +625,15 @@ def _findPlugin(self, plugin_id: str) -> Optional[types.ModuleType]:
                 if current_version > highest_version:
                     highest_version = current_version
                     final_location = loc
+
+        # Move data (if any) to central storage
+        central_storage_file = os.path.join(final_location, plugin_id, "central_storage.json")
+        if os.path.exists(central_storage_file):
+            try:
+                with open(central_storage_file, "r", encoding="utf-8") as file_stream:
```

Comment:
Code style: Spaces around binary operators.
```suggestion
                with open(central_storage_file, "r", encoding = "utf-8") as file_stream:
```